src/OpenSpaceToolkit/Physics/Environment/Gravitational/Earth.cpp

```diff
@@ -45,6 +46,7 @@ const Model::Parameters Earth::WGS84_EGM96 = {
     Length::Meters(6378137.0),
     1.0 / 298.257223563,
     -4.841653717360e-04,
+    9.57254173792e-07,
```

Comment:
I think you should also add it to the EGM96 model, since that's the one we use most often - we almost never use the WGS84_EGM96 model since its a weird hybrid.